### PR TITLE
Fix disable_tagged_api tunable (backport from master)

### PR DESCRIPTION
**Pull request by nirbhayc**
Signed-off-by: Nirbhay Choubey <nchoubey@bloomberg.net>

### CODE REVIEW

**Review by roborivers — COMMENTED** [review]
Lint check:

```diff


@@ cdb2api/cdb2api.c @@
+ Please port the change to cdb2jdbc if necessary.
+ If you add any internal configuration state to this file, please update the reset_the_configuration() function as well to include it.
@@ db/db_tunables.h @@
+ Please update the tunable documentation (docs/pages/config/config_files.md) accordingly.
@@ sqlite/ext/comdb2/tables.c @@
+ If a new system table is added, be sure to document it in docs/pages/programming/system_tables.md.
@@ sqlite/src/parse.y @@
+ Custom tokens and rules must be defined after SQLite's.
+ Please update SQL documentation and bubble diagrams to reflect the changes you've made, if necessary.
@@ sqlite/src/vdbeInt.h @@
+ If the order of members in struct Mem is changed, columnNullValue (defined in sqlite/vdbeapi.c) needs to be reordered accordingly.
@@ sqlite/src/vdbeaux.c @@
+ If a new fixed-length serial type is introduced, sqlite3IsFixedLengthSerialType (defined in sqlite/vdbeInt.h) needs to be changed accordingly.
@@ db/prefault_helper.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ db/sqlglue.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ db/fdb_bend.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ db/handle_buf.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ schemachange/sc_global.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ util/walkback.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
@@ util/logmsg.c @@
+ Please remove debugging statements, and replace fprintf/printf with logmsg.
```

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/formatting-a-pr.html'>Use clang-format</a> or <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/9695d0cb/format.diff'>view diff</a> &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: Success &#10003;.</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/9695d0cb/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/9695d0cb/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2781'>Resubmit</a></p>

File: db/sltdbt.c [hunk 1]
```diff
@@ -243,6 +243,82 @@ int init_opcode_handlers()
     return 0;
 }
 
+static inline int opcode_supported(int opcode)
+{
+    static int last_rej = 0;
+    static unsigned long long rejcnt = 0;
+    int now;
+
+    if (!gbl_rowlocks && !gbl_disable_tagged_api)
```
Comment (riverszhang89) [inline db/sltdbt.c:252]: Why do we need to check for rowlocks?

Comment (nirbhayc) [reply db/sltdbt.c:252]: I guess that because we don't want to support all the tagged operations when rowlocks is enabled.

Comment (nirbhayc) [reply db/sltdbt.c:252]: This part of the patch was copied from R6.

**Review by riverszhang89 — APPROVED** [review]

**Pull request merged**